Doc/whatsnew/3.15.rst

```diff
@@ -286,6 +286,13 @@ that may require changes to your code.
 Build changes
 =============
 
+* Removed implicit fallback to the bundled copy of the ``libmpdec`` library.
+  Now this should be explicitly enabled with :option:`--with-system-libmpdec`
+  set to ``no`` or with :option:`!--without-system-libmpdec`.
+  The :option:`!--without-libmpdec` should be used to permit building
```

Comment:
Let's add a link to --with-libmpdec even though we're talking about the negative form.